### PR TITLE
add _parseSwarmSystemStatus

### PR DESCRIPTION
**Pull request by anandkumarpatel**
- parse swarm formatted data in a more sane way
- add test

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -40,6 +53,70 @@ Swarmerode.prototype.swarmHostExists = function (host, cb) {
...
+  }
+
+  function parseLabels (labelString) {
+    var labelsTokens = labelString.split(',')
+    var labels = {}
+    labelsTokens.forEach(function (labelToken) {
+      var pair = labelToken.split('=').map(function (s) {
+        return s.trim()
+      })
+      labels[pair[0]] = pair[1]
```
Comment (podviaznikov) [inline index.js:111]: are you going to cleanup label name?

Comment (anandkumarpatel) [inline index.js:111]: no cleanup needed?

```
{ 
        executiondriver: 'native-0.2',
        kernelversion: '3.13.0-74-generic',
        operatingsystem: 'Ubuntu 14.04.3 LTS',
        org: '146592',
        storagedriver: 'aufs' 
}
```

Comment (podviaznikov) [inline index.js:111]: nice

**Comment by podviaznikov** [comment]
+1

**Comment by podviaznikov** [comment]
Might be a good idea to add function `doesDockExist`. We reimplement that one in few places

**Comment by podviaznikov** [comment]
Sorry, this library already has one `swarmHostExists`

**Comment by bkendall** [comment]
Good work. Looks good to me! going to be `v0.2.0`

**Pull request merged**